common/src/test/java/tachyon/util/io/FileUtilsTest.java

```diff
@@ -82,6 +88,16 @@ public void moveFileTest() throws IOException {
   }
 
   @Test
+  public void moveNonExistentFileTest() throws IOException {
+    // ghostFile is never created, so deleting should fail
+    File ghostFile = new File(mTestFolder.getRoot(), "ghost.txt");
+    File toFile = mTestFolder.newFile("to.txt");
+    mException.expect(IOException.class);
+    FileUtils.move(ghostFile.getAbsolutePath(), toFile.getAbsolutePath());
+    Assert.assertTrue(ghostFile.delete());
```

Comment:
Actually, on a second thought I realized that we actually don't want line 97 to be evaluated (because the assertion would fail since the ghostFile does not exist). So this pattern is in fact used to assert that the exception occurred.